merkle/log_verifier_test.go

```diff
@@ -118,228 +118,184 @@ var (
 	}
 )
 
-func verifierCheck(v *LogVerifier, leafIndex, treeSize int64, proof [][]byte, root, leafHash []byte) error {
-	// Verify original inclusion proof
-	got, err := v.RootFromInclusionProof(leafIndex, treeSize, proof, leafHash)
-	if err != nil {
-		return err
-	}
-	if want := root; !bytes.Equal(got, want) {
-		return fmt.Errorf("got root:\n%x\nexpected:\n%x", got, want)
-	}
-	if err := v.VerifyInclusionProof(leafIndex, treeSize, proof, root, leafHash); err != nil {
-		return err
-	}
+// inclVerifProbe is a parameter set for inclusion proof verification.
+type inclVerifProbe struct {
```

Comment:
Not sure about the name. What about inclusionProbe? or incProbe? Same for consistency probe.

Reply:
Done.